dev-docs/bidders/trafficgate.md

```diff
@@ -27,8 +29,89 @@ The TrafficGate Bidding adapter requires setup before beginning. Please contact
 ### Bid Params
 
 {: .table .table-bordered .table-striped }
-| Name          | Scope    | Description  | Example     | Type     |
-|---------------|----------|--------------|-------------|----------|
-| `placementId` | required | Placement ID | `'12345'`   | `string` |
-|---------------|----------|--------------|-------------|----------|
-| `host`        | required | Host         | `'example'` | `string` |
+| Name          | Scope    | Description      | Example     | Type     |
+|---------------|----------|------------------|-------------|----------|
+| `placementId` | required | Placement ID     | `'12345'`   | `string` |
+|---------------|----------|------------------|-------------|----------|
+| `host`        | required | Host             | `'example'` | `string` |
+|---------------|----------|------------------|-------------|----------|
+| `customFloor` | optional | Custom Bid Floor | `2.55`      | `number` |
+
+
+### AdUnit Format for Banner
+```javascript
+var adUnits = [{
+    code: 'test-banner-div ',
+    mediaTypes: {
+        banner: {
+            sizes: [[300, 250]],
+        }
+    },
+    bids: [{
+        bidder: 'trafficgate',
+        params: {
+            placementId: '16',
+            host: 'example',
+            test: 1
```

Comment:
Please either remove this `test` parameter or document it in the params table above.